Ingestion Controls - Agent and tracing library version (#15379)
DataDog/documentation: content/en/tracing/trace_pipeline/ingestion_controls.md

For further investigations, use the [APM Trace - Estimated Usage Dashboard][12], which provides global ingestion information as well as breakdown graphs by `service`, `env` and `ingestion reason`.

### Agent and tracing library versions

> View which **Datadog Agent and tracing library versions** your service is using. Compare versions in use to the latest released versions to make sure you’re always running recent and up-to-date Agents and libraries.

Reviewer: ```suggestion
See the **Datadog Agent and tracing library versions** your service is using. Compare the versions in use to the latest released versions to make sure you are running recent and up-to-date Agents and libraries.
```

{{< img src="tracing/trace_indexing_and_ingestion/agent_tracer_version.png" style="width:90%;" alt="Agent and tracing library versions" >}}

> **Note**: You need to upgrade the Datadog Agent to 6.34 or 7.34 for the version information to be reported.

Reviewer: ```suggestion
**Note**: You need to upgrade the Agent to v6.34 or v7.34 for the version information to be reported.
```

### Configure the service ingestion rate

Click **Manage Ingestion Rate** to get instructions on how to configure your service ingestion rate.

{{< img src="tracing/trace_indexing_and_ingestion/service_ingestion_rate_config.png" style="width:100%;" alt="Change the Service Ingestion Rate" >}}

> To specify that a specific percentage of a service's traffic should be sent, add an environment variable or a generated code snippet to your tracing library configuration for that service.

Reviewer: ```suggestion
To specify a specific percentage of a service's traffic to be sent, add an environment variable or a generated code snippet to your tracing library configuration for that service.
```
